aptible/client_utils_test.go

```diff
@@ -69,24 +70,32 @@ func TestGenerateErrorFromClientError(t *testing.T) {
 				},
 			},
 			wantErr:   true,
-			errorBody: "unable to properly decode error (missing fields to properly generate error) status code (400) error message (resource not found)",
+			errorBody: "unable to properly decode error (missing fields to properly generate error) -  status code (400) error message (resource not found)\n",
 		},
 		{
-			name: "return a marshalable error (if server sends back garbled response)",
+			name: "return a marshalable error with a nil payload should break early",
 			args: args{
 				abstractedError: nil,
 			},
 			wantErr:   true,
-			errorBody: "Unable to properly decode error in marshal from client - json: cannot unmarshal string into Go value of type aptible.clientError\n",
+			errorBody: "Error without a valid payload: <nil>\n",
 		},
 		{
-			name: "return a unmarshalable error (but unmarshalable into expected type)",
+			name: "return a unmarshalable error (invalid json)",
```

Comment:
@joshraker i updated the message here which was wrong though, so you did mention an inconsistency i should've cleaned up